File: LeetCode/l1217.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& chips) {
        int c[2]={0};
        for(int i=chips.size()-1;i>=0;i--)c[chips[i]%2]++;
        return min(c[0],c[1]);
    }
};

File: LeetCode/l1217.rs
impl Solution {
    pub fn min_cost_to_move_chips(chips: Vec<i32>) -> i32 {
        let mut c=0;
        for i in 0..chips.len() {
            if(chips[i]%2==0){
                c+=1;
            }
        }
        return core::cmp::min(chips.len() as i32-c,c);
    }
}